content/rc/rc-quickstart.md

```diff
@@ -76,127 +87,135 @@ Once you have the username and password, select **Connect** to open the connecti
 
 The connection wizard provides code snippets to connect to your database with the following programming languages:
 
-- node.js using [node-redis](https://github.com/redis/node-redis#installation)
+- node.js using [node-redis](https://github.com/redis/node-redis/blob/master/README.md)
 - .NET using [StackExchange.Redis](https://stackexchange.github.io/StackExchange.Redis/)
 - Python using [redis-py](https://github.com/redis/redis-py#redis-py)
 - Java using [Jedis](https://github.com/redis/jedis#jedis)
 
+{{<image filename="images/rc/connection-wizard-clients.png" alt="The connection wizard clients." >}}{{< /image >}}
+
 See the [client list](https://redis.io/docs/clients/) to view all Redis clients by language.
 
 #### Code example (Python)
 
 To connect to your database using the [redis-py](https://github.com/redis/redis-py#redis-py) library for Python:
 
-1.  Install the Redis client if it is not already installed.
+1. Install the Redis client if it is not already installed.
 
     ```sh
     $ sudo pip install redis
     ```
 
-2.  In the connection wizard, under **Redis Client**, select Python from the **Select your client** menu.
+2. In the connection wizard, under **Redis Client**, select Python from the **Select your client** menu.
 
 3. Select **Copy** to copy the connection code for your database.
 
-4. Add the copied code to your program and replace `<username>` and `<password>` with your username and password.
+4. Create a python file ```vi example_redis.py```
+
+5. Add the copied code to your Python program.
 
-    ```python
+    ```
     import redis
 
     r = redis.Redis(
-      host='<host>',
-      port=<port>,
-      password='<password>')
+    host='<HOST>',
+    port=<PORT>,
+    password='<PASSWORD>')
+    ```
+   
+6. Add the following code to your Python program.
 
-    # Redis commands
+    ```
     r.set('hello', 'world')
     print(r.get('hello'))
     ```
 
-5. Run the program.
+7. Run the program.
 
     ```sh
     $ python example_redis.py
-    world
+    b'world'
     ```
 
-### RedisInsight{#using-redisinsight}
+### redis-cli (via Docker){#using-rediscli}
 
-RedisInsight is a free Redis GUI that is available for MacOS, Windows, and Linux.
+The [`redis-cli`]({{<relref "/rs/references/cli-utilities/redis-cli/">}}) utility is installed when you install Redis.  It provides a command-line interface that lets you work with your database using core [Redis commands](https://redis.io/commands/).
 
-1. In the connection wizard, under **Redis Client**, select your operating system from the **Download RedisInsight** menu.
+[Docker](https://www.docker.com/) provides a convenient way to run `redis-cli` without the full installation experience.
 
-1. Select **Download** to download RedisInsight.
+Run the following commands to create a Redis Docker container and connect to your database with `redis-cli`:
```

Comment:
@dmaier-redislabs Same with this section (and same ticket - DOC-2510) - do we have somewhere else we talk about installing Redis with docker? redis.io doesn't have anything about docker in the [Install Redis](https://redis.io/docs/install/install-redis/) section.